dev-packages/rollup-utils/npmHelpers.mjs

```diff
@@ -44,8 +43,6 @@ export function makeBaseNPMConfig(options = {}) {
     excludeIframe: undefined,
   });
 
-  const codecovPlugin = makeCodeCovPlugin();
```

Comment:
I'm also removing codecov for now until they fix their reporting for our sdk setup.